<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tibia Market Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/excelent.png</normaloff>:/img/excelent.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbSell">
      <property name="text">
       <string>Sell Offers:</string>
      </property>
      <property name="buddy">
       <cstring>sbSellItems</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="localized_spin_box" name="sbSellOffers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> offers</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="localized_spin_box" name="sbSellItems">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> items</string>
      </property>
      <property name="maximum">
       <number>64000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="localized_spin_box" name="sbSellTotal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> gp</string>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QToolButton" name="btSellNew">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="icon">
       <iconset theme="list-add">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QToolButton" name="btSellDel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="icon">
       <iconset theme="list-remove">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QToolButton" name="btSellEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="document-properties">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QToolButton" name="btSell">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="go-down">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QToolButton" name="btCustomSell">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="go-jump">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="lbBalance">
      <property name="text">
       <string>Balance:</string>
      </property>
      <property name="buddy">
       <cstring>sbBalance</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="localized_spin_box" name="sbBalance">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> gp</string>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="localized_spin_box" name="sbBalanceItems">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> gp)</string>
      </property>
      <property name="prefix">
       <string>(</string>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QToolButton" name="btBalance">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="document-properties">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="14">
     <widget class="market_view" name="tbSell">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbBuy">
      <property name="text">
       <string>Buy Offers:</string>
      </property>
      <property name="buddy">
       <cstring>sbBuyItems</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="localized_spin_box" name="sbBuyOffers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> offers</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="localized_spin_box" name="sbBuyItems">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> items</string>
      </property>
      <property name="maximum">
       <number>64000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="localized_spin_box" name="sbBuyTotal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> gp</string>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QToolButton" name="btBuyNew">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="icon">
       <iconset theme="list-add">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QToolButton" name="btBuy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="go-down">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QToolButton" name="btBuyDel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="icon">
       <iconset theme="list-remove">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QToolButton" name="btCustomBuy">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="go-jump">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QToolButton" name="btBuyEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="document-properties">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="14">
     <widget class="market_view" name="tbBuy">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="10" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbLastTransaction">
        <property name="text">
         <string>Last transaction:</string>
        </property>
        <property name="buddy">
         <cstring>dtLastTransaction</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtLastTransaction">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="9">
     <widget class="QCheckBox" name="cbSync">
      <property name="text">
       <string>Sincronizar</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionRevert"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionReport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="icon">
    <iconset theme="document-print-preview">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="document-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset theme="document-revert">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>localized_spin_box</class>
   <extends>QSpinBox</extends>
   <header>localized_spin_box.h</header>
  </customwidget>
  <customwidget>
   <class>market_view</class>
   <extends>QTableView</extends>
   <header>market_view.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sbSellOffers</tabstop>
  <tabstop>sbSellItems</tabstop>
  <tabstop>sbSellTotal</tabstop>
  <tabstop>btSellNew</tabstop>
  <tabstop>btSellDel</tabstop>
  <tabstop>btSellEdit</tabstop>
  <tabstop>btSell</tabstop>
  <tabstop>btCustomSell</tabstop>
  <tabstop>sbBalance</tabstop>
  <tabstop>sbBalanceItems</tabstop>
  <tabstop>btBalance</tabstop>
  <tabstop>tbSell</tabstop>
  <tabstop>sbBuyOffers</tabstop>
  <tabstop>sbBuyItems</tabstop>
  <tabstop>sbBuyTotal</tabstop>
  <tabstop>btBuyNew</tabstop>
  <tabstop>btBuyDel</tabstop>
  <tabstop>btBuyEdit</tabstop>
  <tabstop>btBuy</tabstop>
  <tabstop>btCustomBuy</tabstop>
  <tabstop>cbSync</tabstop>
  <tabstop>dtLastTransaction</tabstop>
  <tabstop>tbBuy</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
